source/about/self-hosted-subscriptions.rst

```diff
@@ -82,17 +82,29 @@ This process adds additional users to your existing subscription and is not a ne
 
 If you don't have a self-hosted license in the Customer Portal and want to add users, `contact us <https://mattermost.com/contact-us/>`__.
 
-Quarterly report
-^^^^^^^^^^^^^^^^
+Quarterly true-up report
+------------------------
 
-When you subscribe to an annual plan with Mattermost, you agree to provide quarterly reports of the actual number of registered users within your system if you're above the amount you subscribed in your purchase order. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**.
+When you buy an annual Mattermost subscription, you agree to provide quarterly reports of the actual number of registered users within your system. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**.
```

Comment:
Consider changing formatting of "Inactive" from italics to bold, given that the docs are referencing the System Console UI.